src/babylon.scene.ts

```diff
@@ -930,6 +930,14 @@
 
         private _depthRenderer: Nullable<DepthRenderer>;
         private _geometryBufferRenderer: Nullable<GeometryBufferRenderer>;
+        public get geometryBufferRenderer(): Nullable<GeometryBufferRenderer> {
+            return this._geometryBufferRenderer;
+        }
+        public set geometryBufferRenderer(geometryBufferRenderer: Nullable<GeometryBufferRenderer>) {
+            if (geometryBufferRenderer && geometryBufferRenderer.isSupported) {

```

Comment:
I do not understand this setter. Did you forget to set the value?

Reply:
The next line should be 
`this._geometryBufferRenderer = geometryBufferRenderer;`

Reply:
Oh sorry! By accident!